Duplicati/Server/webroot/ngax/scripts/services/AppUtils.js

```diff
@@ -302,6 +314,10 @@ backupApp.service('AppUtils', function($rootScope, $timeout, $cookies, DialogSer
             if (d['--' + k] !== undefined) {
                 DialogService.dialog(gettextCatalog.getString('Error'), gettextCatalog.getString('Duplicate option {{opt}}', { opt: k }));
                 return false;
+            } else if (k == 'include' || k == 'exclude') {
+                // Cannot specify filters in extra options
+                DialogService.dialog(gettextCatalog.getString('Error'), gettextCatalog.getString('Cannot specify filter include or excludes in extra options'));
+                return false;
```

Comment:
I don't see how this can be called, could you please explain this code function ? as far as I can tell, it's not called when using the test-filter called and I don't see how it could ever be called since the filters are not part of extra options, but of the backup object.

Reply:
If I remember correctly this can be triggered when you go to advanced options in the backup settings, edit as text and add `--include=x`, which is why I added this check.
Otherwise someone might add multiple include or exclude lines there, which would be ignored.